docs/how_to_guides/configuring_metadata_stores/how_to_configure_a_validation_result_store_to_postgresql.rst

```diff
@@ -1,18 +1,102 @@
 
-    "Whether I shall turn out to be the hero of my own life, or whether that station will be held by anybody else, these pages must show."
+.. admonition:: Prerequisites: This how-to guide assumes that you have already:
 
+    - Configured a :ref:`Data Context <tutorials__getting_started__initialize_a_data_context>`.
+    - Configured an :ref:`Expectations Suite <tutorials__getting_started__create_your_first_expectations>`.
+    - Configured a :ref:`Checkpoint <tutorials__getting_started__set_up_your_first_checkpoint>`.
+    - Configured a `PostgreSQL <https://www.postgresql.org/>`_ database with appropriate credentials.
 
-This guide is a stub. We all know that it will be useful, but no one has made time to write it yet.
 
-If it would be useful to you, please comment with a +1 and feel free to add any suggestions or questions below.
+1. **Configure the** ``config_variables.yml`` **file with your database credentials**
+
+    We recommend that database credentials be stored in the  ``config_variables.yml`` file, which is located in the ``uncommitted/`` folder by default, and is not part of source control.  The following lines add database credentials under the key ``db_creds``. Additional options for configuring the ``config_variables.yml`` file or additional environment variables can be found `here. <https://docs.greatexpectations.io/en/latest/how_to_guides/configuring_data_contexts/how_to_use_a_yaml_file_or_environment_variables_to_populate_credentials.html>`_
+
+    .. code-block:: yaml
+
+        db_creds:
+            drivername: postgres
+            host: '<your_host_name>'
+            port: '<your_port>'
+            username: '<your_username>'
+            password: '<your_password>'
+            database: '<your_database_name>'
+
+
+2. **Identify your Data Context Validations Store**
+
+    In your ``great_expectations.yml``, look for the following lines.  The configuration tells Great Expectations to look for Validations in a store called ``validations_store``. The ``base_directory`` for ``validations_store`` is set to ``uncommitted/validations/`` by default.
+
+
+    .. code-block:: yaml
+
+        validations_store_name: validations_store
+
+        stores:
+            validations_store:
+                class_name: ValidationsStore
+                store_backend:
+                    class_name: TupleFilesystemStoreBackend
+                    base_directory: uncommitted/validations/
+
+3. **Update your configuration file to include a new store for Validations on PostgreSQL**
+
+    In our case, the name is set to ``expectations_postgres_store``, but it can be any name you like.  We also need to make some changes to the ``store_backend`` settings.  The ``class_name`` will be set to ``DatabaseStoreBackend``, and ``credentials`` will be set to ``${db_creds}``, which references the corresponding key in the ``config_variables.yml`` file.
+
+    .. code-block:: yaml
+
+        expectations_store_name: validations_postgres_store
```

Comment:
Should this be `validations_store_name`?

Reply:
you're absolutely right